HW_module_6.py

```diff
@@ -0,0 +1,155 @@
+        message = f'Private Ad -----------------------\n{self.adv_message}\n' \
+                  f'Actual until: {self.actual_until},' \
+                  f'{(datetime.datetime.strptime(self.actual_until, "%d/%m/%y") - datetime.datetime.now()).days}days left\n' \
+                  f'----------------------------------\n\n'
+        message = normalize(message)
+        self.ptf = PrintMessage(message)
+        ptf = self.ptf
+        ptf.print_message()
+
+
+class Guess:
+    def __init__(self, guessing):
+        self.guessing = guessing
+
+    # create an interactive method with indicating possibility using random elements from list
+    def ask_future(self):
+        test_list = ['It is certain', 'It is decidedly so', 'Without a doubt', 'Yes – definitely', 'As I see it, yes',
+                        'Most likely', 'Ask again later', 'Better not tell you now', 'Cannot predict now',
+                        'Don\'t count on it', 'My reply is no', 'My sources say no', 'Very doubtful']
+        random_test_list = random.randint(0, len(test_list) - 1)
+        prediction = f'Ask me about your future? --------\n' \
+                     f'Your question - "{self.guessing}",\n' \
+                     f'Witch\'s answer will be - {test_list[random_test_list]}\n' \
+                     f'----------------------------------\n\n'
+        prediction = normalize(prediction)
+        self.prt = PrintMessage(prediction)
+        prt = self.prt
+        prt.print_message()
+
+
+class FromAnotherSource:
+
+    # create a method which makes another directories and reads a file from that directory OR use the default one
+    def read_file(self):
+        original_path = os.path.dirname(os.path.realpath(__file__))
+        print(f'Your default directory is "{os.getcwd()}"')
+        is_true = True
+        while is_true:
+            answer = int(input(
+                f'If you want to ingest your file from default directory - enter 1, if you want to change it - enter 2: '))
+            if answer == 1:
+                while True:
+                    try:
+                        file_name = input('Enter the file name with its format: ')
+                        with open(file_name, "r", encoding='utf-8') as source:
+                            f_contents = source.readlines()
+                        print('Okay, such file exists')
+                        path_for_remove = str(file_name)
+                        is_true = False
+                        break
+                    except FileNotFoundError:
+                        print('No file with such name. Try again')
+                    except PermissionError:
+                        print('No file with such name. Try again')
+            elif answer == 2:
+                while True:
+                    try:
+                        change_path = input('Enter the file path: ')
+                        if not os.path.isdir(change_path):
+                            os.mkdir(change_path)
```

Comment:
Do we actually need this logic here? If I specify a path, which does not exist then it's unlikely this path will contain the files for processing.